hedera-node/hedera-app/src/main/java/com/hedera/node/app/tss/handlers/TssUtils.java

```diff
@@ -0,0 +1,123 @@
+        // FUTURE: This private key must be loaded from disk
+        builder.withSelf(
+                selfNodeId,
+                new PairingPrivateKey(
+                        new FakeFieldElement(BigInteger.valueOf(10L)), SignatureSchema.create(new byte[] {1})));
+        for (var rosterEntry : roster.rosterEntries()) {
+            final int numSharesPerThisNode =
+                    computedShares.get(rosterEntry.nodeId()).intValue();
+            // FUTURE: Use the actual public key from the node
+            final var pairingPublicKey = new PairingPublicKey(
+                    new FakeGroupElement(BigInteger.valueOf(10L)), SignatureSchema.create(new byte[] {1}));
+            builder.withParticipant((int) rosterEntry.nodeId(), numSharesPerThisNode, pairingPublicKey);
+        }
+        // FUTURE: Use the actual signature schema
+        return builder.build(SignatureSchema.create(new byte[] {1}));
+    }
+
+    /**
+     * Validate TSS messages using the TSS library. If the message is valid, add it to the list of valid TSS messages.
+     *
+     * @param tssMessages             list of TSS messages to validate
+     * @param tssParticipantDirectory the participant directory
+     * @return list of valid TSS messages
+     */
+    public static List<TssMessageTransactionBody> validateTssMessages(
+            @NonNull final List<TssMessageTransactionBody> tssMessages,
+            @NonNull final TssParticipantDirectory tssParticipantDirectory,
+            @NonNull final TssLibrary tssLibrary) {
+        final var validTssMessages = new LinkedList<TssMessageTransactionBody>();
+        for (final var op : tssMessages) {
+            final var isValid = tssLibrary.verifyTssMessage(
+                    tssParticipantDirectory, new TssMessage(op.tssMessage().toByteArray()));
+            if (isValid) {
+                validTssMessages.add(op);
+            }
+        }
+        return validTssMessages;
+    }
+
+    /**
+     * Get the TSS messages from the list of valid TSS Message bodies.
+     *
+     * @param validTssOps list of valid TSS message bodies
+     * @return list of TSS messages
+     */
+    public static List<TssMessage> getTssMessages(List<TssMessageTransactionBody> validTssOps) {
+        return validTssOps.stream()
+                .map(TssMessageTransactionBody::tssMessage)
+                .map(k -> new TssMessage(k.toByteArray()))
+                .toList();
+    }
+
+    /**
+     * Compute the number of shares each node should have based on the weight of the node.
+     *
+     * @param rosterEntries         the list of roster entries
+     * @param maxTssMessagesPerNode the maximum number of TSS messages per node
+     * @return a map of node ID to the number of shares
+     */
+    public static Map<Long, Long> computeNodeShares(
+            @NonNull final List<RosterEntry> rosterEntries, final long maxTssMessagesPerNode) {
```

Comment:
`maxSharesPerNode` is the param passed in from `computeTssParticipantDirectory`, and what I expected to see as the name for `maxTssMessagesPerNode`. Are these the same thing?

Reply:
Yes Typo. Changed to `maxSharesPerNode`